docs/GettingStarted.md

```diff
@@ -0,0 +1,135 @@
+# Getting Started
+
+This guide walks through how to get up and running with the AEP Android SDK with only a few lines of code.
+
+> Existing ACP SDK customers should check out the [migration document](./Migration.md).
+
+## Set up a Mobile Property
+
+Set up a mobile property as outlined in the Adobe Experience Platform [docs](https://developer-stage.adobe.com/client-sdks/documentation/getting-started/create-a-mobile-property/)
+
+## Get the Experience Platform SDK
+
+If you cannot access the Mobile Install Instructions dialog box in the Data Collection UI, complete the following sections to get the Adobe Experience Platform SDK. If you already completed the steps in the Mobile Install Instructions dialog box, no need to complete these steps.
+
+## Installation instructions
+
+1. Each extension needs to be added as a dependency to the mobile application project. The following examples will add the Mobile Core, Lifecycle, Identity, Signal and Profile extensions.
+
+```java
+implementation 'com.adobe.marketing.mobile:userprofile:2.0'
+implementation 'com.adobe.marketing.mobile:core:2.0'
+implementation 'com.adobe.marketing.mobile:identity:2.0'
+implementation 'com.adobe.marketing.mobile:signal:2.0'
+implementation 'com.adobe.marketing.mobile:lifecycle:2.0'
+```
+
```

Comment:
Should we add a note asking implementers to check the latest version of dependancies ?